examples/example_mnist.py

```diff
@@ -78,25 +79,25 @@
     x_train.shape[1:-1] + (1,), name="input")
 x = QConv2D(
     32, (2, 2), strides=(2,2),
-    kernel_quantizer=quantized_bits(4,0,1),
-    bias_quantizer=quantized_bits(4,0,1),
+    kernel_quantizer=quantized_linear(4,0,1),
```

Comment:
For now we will probably not replace quantized_bits with quantized_linear due to lack of test in hardware flow, please add specific tests for quantized_linear instead of rewrite unit-tests from quantized_bits.

Reply:
Reverted all example files and notebooks to their former state. No mention of quantized_linear.